docs/pages/server-access/getting-started.mdx

```diff
@@ -0,0 +1,347 @@
+
+   Add an `A` record in your domain registrar entry for the `tele` subdomain and map the first IP address to the `tele` subdomain.
+
+   <Figure
+     align="center"
+     bordered
+     caption="Subdomain and IP mapping"
+   >
+     ![Subdomain and IP mapping](../../img/server-access/subdomains.png)
+   </Figure>
+
+   Your second instance will be a private resource. These are the ports you'll want to initially open:
+
+   | Port | Service | Description |
+   | - | - | - |
+   | 3023 | Proxy | SSH port clients connect to. A proxy will forward this connection to port `#3022` on the destination Node. |
+   | 22 | Cloud Provider | Used to initially access, configure, and provision your cloud instances. We'll configure and launch our instances then demonstrate how to use the `tsh` tool and Teleport in SSH mode thereafter. |
+
+2. Install Teleport on each instance.
+
+   <Admonition type="warning" title="Warning">
+    The examples below may include the use of the `sudo` keyword to make following each step easier when creating resources from scratch. However, we generally discourage the use of `sudo` in real-world production environments per the *Principle of Least Privilege* (POLP). 
+   </Admonition>
+
+   <Tabs>
+     <TabItem label="Amazon Linux 2/RHEL (RPM)">
+      ```bash
+      sudo yum-config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
+      sudo yum install teleport
+
+      # Optional:  Using DNF on newer distributions
+      # $ sudo dnf config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
+      # $ sudo dnf install teleport
+      ```
+     </TabItem>
+
+     <TabItem label="Debian/Ubuntu (DEB)">
+      ```bash
+      curl https://deb.releases.teleport.dev/teleport-pubkey.asc | sudo apt-key add -
+      sudo add-apt-repository 'deb https://deb.releases.teleport.dev/ stable main'
+      sudo apt-get update
+      sudo apt-get install teleport
+      ```
+     </TabItem>
+
+     <TabItem label="Linux">
+      ```bash
+      curl -O https://get.gravitational.com/teleport-v(=teleport.version=)-linux-amd64-bin.tar.gz
+      tar -xzf teleport-v(=teleport.version=)-linux-amd64-bin.tar.gz
+      cd teleport
+      sudo ./install
+      ```
+     </TabItem>
+
+     <TabItem label="ARMv7 (32-bit)">
+      ```bash
+      curl -O https://get.gravitational.com/teleport-v(=teleport.version=)-linux-arm-bin.tar.gz
+      tar -xzf teleport-v(=teleport.version=)-linux-arm-bin.tar.gz
+      cd teleport
+      sudo ./install
+      ```
+     </TabItem>
+
+     <TabItem label="ARMv8 (64-bit)">
+      ```bash
+      curl -O https://get.gravitational.com/teleport-v(=teleport.version=)-linux-arm64-bin.tar.gz
+      tar -xzf teleport-v(=teleport.version=)-linux-arm64-bin.tar.gz
+      cd teleport
+      sudo ./install
+      ```
+     </TabItem>
+   </Tabs>
+
+3. Configure Teleport on the *Bastion Host*.
+
+   Teleport will now automatically acquire an ACME-compliant TLS certificate. 
```

Comment:
Nit: the certificate itself is not `ACME-compliant`, it's just a usual X.509 web certificate teleport gets through ACME protocol. So may be change this to `Teleport will now automatically acquire an X.509 certificate using ACME protocol`

Reply:
Thanks - that could have been improved. The IETF https://datatracker.ietf.org/doc/html/rfc8555:

```
The guiding use case for ACME is obtaining certificates for websites.
```

I had intended this to mean compliant with ACME itself per:

```
Let's Encrypt uses the ACME protocol to verify that you control a given domain name and to issue you a certificate. To get a Let's Encrypt certificate, you'll need to choose a piece of ACME client software to use. 
```

Corrected.